alphacsc/online_dictionary_learning.py

```diff
@@ -121,6 +116,8 @@ def _ensure_fit_init(self, X):
         self.n_channels_ = n_channels = X.shape[1]
 
         self.constants['n_channels'] = n_channels
+        self.constants['XtX'] = X.ravel().dot(X.ravel())
```

Comment:
Actually, the constant `XtX` should be updated as done for the other constants so:
```suggestion
        self.constants['XtX'] = 0
```